### ISSUE TITLE
Returned error empty message

### ISSUE THREAD

**Issue by DanielKillenberger** [thebigredgeek/apollo-errors#39]
I can't seem to figure out why the message I define as a parameter for createError is not being returned when instantiating a new error.
But when I instantiate the error with a message the message gets returned as expected.

E.g. the following error: 

```
const { createError } = require('apollo-errors');

const InvalidCredentialsError = createError('InvalidCredentialsError', {
  message: 'The provided credentials are invalid.'
});

module.exports = { InvalidCredentialsError };
```

I instantiate the error as follows:

```
const valid = await bcrypt.compare(password, user.password);
    if (!valid) {
      throw new InvalidCredentialsError();
    }

```

The resulting response is this (notice the empty message):

```
{
  "data": null,
  "errors": [
    {
      "message": "",
      "name": "InvalidCredentialsError",
      "time_thrown": "2018-04-08T07:59:39.258Z",
      "data": {}
    }
  ]
}

```

But when I instantiate the error like so:

```
const valid = await bcrypt.compare(password, user.password);
    if (!valid) {
      throw new InvalidCredentialsError({ message: 'Testmessage' });
    }
```

I get the following (expected) response:

```
{
  "data": null,
  "errors": [
    {
      "message": "Testmessage",
      "name": "InvalidCredentialsError",
      "time_thrown": "2018-04-08T08:00:40.677Z",
      "data": {}
    }
  ]
}
```
Not sure if it matters but the formatting options I set up as follows:

```
const { formatError } = require('apollo-errors');
const { GraphQLServer, Options } = require('graphql-yoga');
const { Prisma } = require('prisma-binding');
const { makeExecutableSchema } = require('graphql-tools');
const { importSchema } = require('graphql-import');
const resolvers = require('./resolvers');
const directiveResolvers = require('./directiveResolvers');

const schema = makeExecutableSchema({
  typeDefs: importSchema('./src/schema.graphql'),
  resolvers,
  directiveResolvers
});

const options = {
  formatError
};

const server = new GraphQLServer({
  schema,
  context: req => ({
    ...req,
    db: new Prisma({
      typeDefs: 'src/generated/prisma.graphql',
      endpoint: process.env.PRISMA_ENDPOINT,
      secret: process.env.PRISMA_SECRET,
      debug: true
    })
  })
});

server.start(options, () => console.log('Server is running on http://localhost:4000'));
```

Any guidance on how to fix this would be greatly appreciated.

**Comment by kyle-mccarthy** [comment]
I'm experiencing this issue as well with an apollo and koa set up.  If I pass a message to the error like `throw new FooError({ message: 'some message' })` the message will show, otherwise it is an empty string.

**Comment by thebigredgeek** [comment]
Closing this as a duplicate of https://github.com/thebigredgeek/apollo-errors/issues/39

**Comment by DanielKillenberger** [comment]
Is the closing of this issue a mistake?
Especially as the issue has been marked as a duplicate of itself.

**Comment by thebigredgeek** [comment]
Sorry, wrong URL.  No, closing was not a mistake.  This should be fixed in `1.8`.  If you are still having the issue, can you reply here?

**Comment by DanielKillenberger** [comment]
1.8 fixed it!
Thanks :)
